data/hails.txt

```diff
@@ -2030,7 +2030,232 @@ phrase "friendly author"
 
 phrase "friendly disabled"
 	word
-		"Our ship has been disabled! Please come board our ship and patch us up!"
+		"Our ship has been disabled! "
+		"We've gotten ourselves in a bit of a situation! "
+		"We've lost control of our ship! "
+		"Our ship took quite the pounding! "
+		"Hey! "
+		"S.O.S! S.O.S! "
+	word
+		"Please come board our ship and patch us up!"
+		"We would appreciate any help getting us running again!"
+		"We can't fix this our own!"
+		"Could you come over and help us please?"
+		"Requesting emergency repairs!"
+		
+phrase "friendly disabled"
```

Comment:
the possibilities of this phrase can be like
"Hey, Our ship is disabled, "
^ capital letter after a comma, no
also, the first AND second word can be empty, creating "could you lend a hand?" which has no capital letter at all
go through these hails and make sure this stuff doesn't happen